src/ts/mobilev3playerapi.ts

```diff
@@ -0,0 +1,39 @@
+import { PlayerAPI, PlayerEvent, PlayerEventBase, PlayerEventCallback } from 'bitmovin-player';
+import { WrappedPlayer } from './uimanager';
+
+export enum MobileV3PlayerEvent {
+  SourceError = 'sourceerror',
+  PlayerError = 'playererror',
+  PlaylistTransition = 'playlisttransition',
+}
+
+export interface MobileV3PlayerErrorEvent extends PlayerEventBase {
+  name: 'onPlayerError';
```

Comment:
Why do you need the `name` property here?

Reply:
It was defined here https://github.com/bitmovin/player-api-specification/blob/master/ios/events.md
```java
public final class PlayerErrorEvent: PlayerEvent {
    public let name = "onPlayerError"
    public let code: PlayerError.Code
    public let message: String
    public let timestamp: TimeInterval
    public let data: DeficiencyData?
}
```

That's why we used in the `defaultMobileV3ErrorMessageTranslator`